### DISCUSSION TITLE
How to avoid  Failed to parse field directive from error when using native python typing?

### DISCUSSION THREAD

**Discussion by ssbarnea** [mkdocstrings/mkdocstrings#537] [Q&A] [answered]
As we are using python native type support it makes no sense to include the type of the arguments in sphinx docstrings as it would only be redundant.

How can we avoid an error like below:
```
WARNING  -  griffe: source:60: Failed to parse field directive from ':param: foo: ...'
````

When running mkdocs in strict mode this warning becomes a fatal error.

Is there an option we can use to avoid this from happening or another docstring format that can make use of python types so we avoid the redundancy?

If did a search for [docstring_options](https://github.com/search?q=repo%3Amkdocstrings%2Fmkdocstrings+docstring_options&type=issues) but I only found one option related to init, so my guess is that is not yet possible. Any workarounds?

Related to https://github.com/mkdocstrings/griffe/issues/125

**Comment by pawamoy** [comment, marked as answer]
Don't you need to write `:param foo:` instead of `:param: foo:`? The parameter name is still needed. The type  is not even specified in that directive, it's specified with the `:type:` directive IIRC.
